app/templates/static/layout.static.pug

```diff
@@ -5,11 +5,11 @@ html(lang='en')
     meta(charset='utf-8')
     meta(name='viewport', content='width=1024')
     meta(http-equiv='Accept-CH', content='Viewport-Width')
-    //- title CodeCombat - Learn how to code by playing a game
     title(data-i18n="new_home.title")
-    meta(data-i18n="[content]new_home.meta_description")
-    meta(property='og:title', content='CodeCombat: Learn to Code by Playing a Game')
-    meta(property='og:url', content='http://codecombat.com')
+    meta(name="keywords", data-i18n="[content]new_home.meta_keywords")
+    meta(name="description", data-i18n="[content]new_home.meta_description")
+    meta(property='og:title', data-i18n="[content]new_home.title")
+    meta(property='og:url', data-i18n="[content]new_home.meta_og_url")
     meta(property='og:type', content='game')
     meta(property='og:image', content='http://codecombat.com/images/pages/home/play_img.png')
```

Comment:
Anyone know what does this field used for?

Reply:
since there are twitter tag below, og tag may mainly used in facebook link. Maybe it works on some url2preview tools like [onebox](https://github.com/discourse/onebox) , too.
![image](https://user-images.githubusercontent.com/11417632/69245261-795e6d00-0be1-11ea-87a1-9477e91e5c16.png)
![image](https://user-images.githubusercontent.com/11417632/69245280-824f3e80-0be1-11ea-976a-f98f10be3fdd.png)